app/models/message.rb

```diff
@@ -32,11 +32,12 @@ class Message < ApplicationRecord
 
   validates :sender_id, presence: true
   validates :type, :origin, length: { maximum: 255 }
+  validates :subject, presence: true
 
   after_save :update_recipients
   after_initialize :default_values
 
-  attr_protected :sender_id, :tenant_id
+  attr_protected :tenant_id
```

Comment:
I think this is dangerous. Although this issue is present in the previous implementation also.
Basically, by passing `message[sender_id]` in a form, the user can send messages as if they were from another account. I think we need to discard this field, and set it through the `current_account`.

Reply:
Well, I mean that we should not permit `sender_id` to be taken from the form data (i.e. coming from the user).

Reply:
So we have made it attr_protected